bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/thing/meter_device_emucs_v1_0.xml

```diff
@@ -17,6 +17,10 @@
 			<channel id="p1_emucs_version_output" typeId="p1VersionType"/>
 			<channel id="p1_timestamp" typeId="p1TimestampType"/>
 		</channels>
+		<properties>
+			<property name="thingTypeVersion">1</property>
+		</properties>
+
```

Comment:
I believe you added this property to the wrong thing type. It should be added to thing id `electricity_emucs_v1_0` in file `meter_electricity_emucs_v1_0.xml`.

Reply:
Done